configs/webpack.config.js

```diff
@@ -0,0 +1,45 @@
+const path = require('path');
+const webpack = require('webpack');
+const defaultConfig = require('@wordpress/scripts/config/webpack.config');
+const ESLintWebpackPlugin = require('eslint-webpack-plugin');
+const StyleLintPlugin = require('stylelint-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const eslintConfig = require('./eslint.js');
+const styleLintConfig = require('./stylelint.js');
+
+/** @type {webpack.Configuration} */
+const webpackConfig = {
+  ...defaultConfig,
+  plugins: [
+    ...defaultConfig.plugins,
+    new ESLintWebpackPlugin({
+      baseConfig: eslintConfig,
+    }),
+    new StyleLintPlugin({
+      config: styleLintConfig,
+    }),
+    new CopyWebpackPlugin({
+      patterns: [
+        {
+          from: `static/**/*`,
+          context: 'src',
+          noErrorOnMissing: true,
+        },
+      ],
+    }),
+  ],
+  resolve: {
+    ...defaultConfig.resolve,
+    alias: {
+      ...defaultConfig.resolve.alias,
+      '@Components': path.resolve(process.cwd(), 'src/components'),
+      Components: path.resolve(process.cwd(), 'src/components'),
+      '@Static': path.resolve(process.cwd(), 'src/static'),
+      Static: path.resolve(process.cwd(), 'src/static'),
+      '@Utils': path.resolve(process.cwd(), 'src/utils'),
+      Utils: path.resolve(process.cwd(), 'src/utils'),
```

Comment:
Need some clarification here. With this being a [`process.cwd()`](https://nodejs.org/docs/latest/api/process.html#process_process_cwd) this is only going to look for alias' with the directory where node was invoked from, so depending on the multi-project setup will attempt to alias from root. Is this unaffected due to the concurrency change else-where?

Reply:
`concurrently` will first `cd` into the project directory then run `wp-scripts` with our webpack config, so I think this should work.

However, directory based functionality is something that's definitely subject to change throughout this PR, as I try to understand exactly how things should behave in single vs multiple project mode.